numba/tests/test_ssa.py

```diff
@@ -240,6 +241,7 @@ def foo(pred):
 
         self.check_func(foo, False)
 
+    @unittest.skip("Fails due to reshape not supporting A layout")
```

Comment:
I'm concerned that this change, in effect, is causing a regression. This test was added based on support for user code and so not supporting this case would potentially break existing code.